ci/azure-pipelines.yml

```diff
@@ -44,20 +44,28 @@ jobs:
         displayName: Publish Documentation Artifacts
 
   - job: UnitTests
-    condition: eq(dependencies.VerifyBuild.outputs['SetJobTriggers.runTests'], 'true')
-    dependsOn: VerifyBuild
+#    condition: eq(dependencies.VerifyBuild.outputs['SetJobTriggers.runTests'], 'true')
+#    dependsOn: VerifyBuild
     pool:
       vmImage: ubuntu-18.04
     steps:
       - template: install_deps.yml
+      - script: cat /proc/sys/fs/file-max
+      - script: whoami
+      - script: sudo cat /etc/security/limits.conf
+      - script: ulimit -Sn
+      - script: ulimit -Hn
```

Comment:
you may need to run `su - vsts` to get a new session so the new limits take effect

Reply:
Looks like you approved this change on release-1.4, so marking this resolved.